tests/connector-service-tests/test/apitests/connector_test.go

```diff
@@ -554,6 +566,33 @@ func certificateRotationSuite(t *testing.T, tokenRequest *http.Request, skipVeri
 		require.Nil(t, errorResponse)
 	})
 
+	t.Run("should renew client certificate using information from management info endpoint", func(t *testing.T) {
```

Comment:
This code is almost identical to "should renew client certificate using information from management info endpoint". Please consider adding some function to make code shorter.

Reply:
I merged both tests into one. It uses management info response to fetch the certificate data. As far as I know it will be the default flow. Value is checked if it equals the one from csrInfo response. I think this is the cleanest way to perform such a test